__tests__/api.test.js

```diff
@@ -160,7 +161,7 @@ describe("GET /api/users", () => {
       .then(({ body }) => {
         expect(Array.isArray(body.users)).toBe(true);
         expect(body.users.length > 0).toBe(true);
-        expect(body.users.length).toBe(4)
+        expect(body.users.length).toBe(4);
 
         body.users.forEach((obj) => {
```

Comment:
I'd make sure you're branching from main.
Otherwise you have stale code on a new branch, then have to change it in two places. 
It sort of makes the branching/PR idea redundant.